infrastructure/README.md

```diff
@@ -0,0 +1,43 @@
+# Infrastructure
+
+Multiple deployment pipelines for Forseti Visualizer:
+
+1. Docker on GCE with COS
+2. Docker on GKE with COS
+3. Docker on Cloud Run
+
+## Overview
+
+* Each script should use Cloud Build to build the docker image and push it to the local gcr.io repository.  
+## Getting Started
+
+```bash
+# replace the variable (PROJECT_ID) at the top of the build-images.sh file
+./build-images.sh
+
+# create a file named "source.env"
+cat > source.env << EOF
+export CLOUDSQL_HOSTNAME="1.2.34.56"
+export CLOUDSQL_USERNAME="user"
+export CLOUDSQL_PASSWORD="password"
+export CLOUDSQL_SCHEMA="forseti_security"
+export FORSETI_SERVER_VM_CHANNEL="1.2.3.4:50051"
+export FORSETI_DATA_MODEL_HANDLE="abcdaa3a5bc9fd9acfaf50fdd3620534"
```

Comment:
Seems like we need to manually export the model handle, is this a manual step user needs to do everytime they want to update w/ the latest inventory?

Reply:
At this time, yes, otherwise, it will be sourcing explain data from an older version.